common/arch/win32/rbaudio.cpp

```diff
@@ -307,7 +310,8 @@ int RBAPauseResume()
         con_puts(CON_VERBOSE, "RBAudio win32/MCI: Toggle Playback pause");
         RBAPause();
     }
-    else if (mciStatusParms.dwReturn == MCI_MODE_PAUSE)
+        // MCI_MODE_STOP is actually paused, because logic
```

Comment:
`because logic` is a bit vague.  Could you elaborate on this, either inline or in a block comment that explains the overall design?

Reply:
This is an MCi design element which this code can't do anything about. The comment could probably be more clear on that, though.